nautobot/docs/additional-features/job_scheduling_and_approvals.md

```diff
@@ -0,0 +1,59 @@
+# Job Scheduling and Approvals
+
+Oftentimes jobs will need to be run at a later date or periodically, or require approval from someone before they can be started. To this end, Nautobot offers facilities for scheduling and approving jobs.
+
+## Job Scheduling
+
+Jobs can be scheduled to be run immediately, at some point in the future, or at an interval.
+
+Jobs can be scheduled through the UI or the API.
+
+### Scheduling via the UI
+
+The Job Scheduling views can be accessed via the navigation at `Extensibility > Jobs`, selecting a Job as appropriate.
+
+The UI allows you to select a scheduling type. Further fields will be displayed as appropriate for that schedule type.
+
+If the job requires no approval, it will then be added to the queue of scheduled jobs or run immediately. Otherwise, the job will be added to the approval quere where it can be approved by other users.
```

Comment:
```suggestion
If the job requires no approval, it will then be added to the queue of scheduled jobs or run immediately. Otherwise, the job will be added to the approval queue where it can be approved by other users.
```